src/actions/router.js

```diff
@@ -38,10 +38,6 @@ export function goTo(pathOrUrl, options = {}) {
 
 export function goToMessage(message) {
   return dispatch => {
-    dispatch({
```

Comment:
This was not used anywhere

Reply:
@nikgraf in some actions we use empty dispatch, just to see that current action fired from components. It's easier to debug, then